pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -2164,6 +2167,9 @@ func testProtocolClassificationLinux(t *testing.T, tr *Tracer, clientHost, targe
 // Wraps the call to the Go-TLS attach function and waits for the program to be traced.
 func goTLSAttachPID(t *testing.T, pid int) {
```

Comment:
if we have `pause` and `resume` mechanism, we don't need attack-detach mechanism
we implemented it as a workaround